assets/js/search.js

```diff
@@ -39,62 +39,66 @@
   /**
    * Function displays the total items displayed on the page
    */
-  function showItemsCount(searchEntry, totalCount, numSubmodules) {
-    if (searchEntry.type === 'libraryEntries') {
+  function showItemsCount(totalCount, numSubmodules) {
+    if ($('.table-clickable-row'.length > 0)) {
```

Comment:
This looks like a bug... Shouldn't it be `$('.table-clickable-row').length > 0`? That is, check the paren placement.

Reply:
Definitely a bug. Quite surprised why it worked without throwing any errors. I'll fix it